pkg/ast/spl/spl.peg

```diff
@@ -505,6 +442,170 @@ Start <- SPACE? initialSearch:(InitialSearchBlock) filterBlocks:(FilterBlock)* q
 
     return q, nil
 }
+/ SPACE? PIPE CMD_GENTIMES SPACE genTimesOption:(GenTimesOptionList) queryAggBlocks:(QueryAggergatorBlock)* SPACE? EOF {
```

Comment:
So the search command will not be there at all when `gentimes` is used?

Reply:
No, gentimes has to be the first command in the search pipeline.